rules_docker/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
```

Comment:
I think the better test would be the one in examples/ (similar to examples/go) because that will use federation "from the outside". wdyt @laurentlb ?

Reply:
It seems the content in rules_go/BUILD is very similar to examples/go/
Let me know if you'd rather I move the example to examples/docker/ (or something like that)

Reply:
moved content to examples/docker. However I find it strange that the examples folder is in the bazelingore file (https://github.com/bazelbuild/bazel-federation/blob/master/.bazelignore)

Reply:
Ah sorry for being unclear, I meant a different thing:
- examples/go have their own [WORKSPACE](https://github.com/bazelbuild/bazel-federation/blob/master/examples/go/WORKSPACE) that uses federation as an external dependency - the ways federation clients are supposed to use it.  

That is why `examples` are ignored: they do not belong to a federation workspace, but each example defines its own workspace, depending on federation.

Does this make sense?